neuro_san/service/http/handlers/base_request_handler.py

```diff
@@ -63,7 +59,6 @@ def initialize(self,
         """
 
         self.agent_policy = agent_policy
-        self.agents_updater = agents_updater
         self.forwarded_request_metadata: List[str] = forwarded_request_metadata
```

Comment:
No separate "agents updating" anymore - http server state is updated by listening to changes in network_storage, the same as for gRPC server.